terraform/modules/fargate-service/ecs.tf

```diff
@@ -0,0 +1,46 @@
+resource "aws_ecs_service" "service" {
+  depends_on                         = [aws_lb_listener.listener]
+  name                               = "${var.name_prefix}-service"
+  cluster                            = var.ecs_cluster_arn
+  task_definition                    = var.task_definition_arn
+  launch_type                        = "FARGATE"
+  desired_count                      = var.desired_count
```

Comment:
The desired_count should be removed and a ignore_changes lifecycle hook should be added for it because it keeps breaking pipelines when autoscaling kicks in

Reply:
Done 👍